### DISCUSSION TITLE
Slow performance

### DISCUSSION THREAD

**Discussion by Lauta11** [nextcloud/all-in-one#2523] [Questions] [answered]
When I do a connection test to my nextcloud of approximately 100 connections to the server, it begins to saturate the resources.
The resource the my pc is 8 CPU, 12 GB RAM.
The settings I currently have are:
pm = ondemand
pm.max_children = 235
pm.start_servers = 2
pm.min_spare_servers = 1
pm.max_spare_servers = 3
pm.process_idle_timeout = 15s
These configurations are displayed from /usr/local/etc/php-fpm.d/www.conf
If I modify the parameter pm.max_children I have the feeling that it is not limiting them because it exceeds said limit.
![imagen](https://github.com/nextcloud/all-in-one/assets/88099919/850c9742-6be4-48b2-a8d2-9ea070552a29)
I really need it to work with 300-500 simultaneous users.

I hope you can help me. Thank you

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1335

**Comment by szaimen** [comment]
@Lauta11 btw, for that amount of users you should consider getting an enterprise subscription. See https://nextcloud.com/all-in-one
